tests_config/require_standalone.js

```diff
@@ -0,0 +1,22 @@
+const prettier = require('prettier/standalone');
+const solidityPlugin = require('../src/index');
+
+module.exports = {
+  formatWithCursor(input, options) {
+    const $$$options = {
```

Comment:
`$$$options` can you tell me more about it?

Reply:
Just a name to be different from `options` it came from the original file. That one instead of importing the code, it reads the files and treats it as a string to be evaluated by the `vm`. I didn't have much time to go through this process so there's some room for improvement here.